scripts/lib.sh

```diff
@@ -127,12 +127,7 @@ install-py-dev() {
 patch-selenium() {
     # XXX: getsentry repo calls this!
     # This hack is until we can upgrade to a newer version of Selenium
-    fx_profile=.venv/lib/python3.8/site-packages/selenium/webdriver/firefox/firefox_profile.py
-    # Remove this block when upgrading the selenium package
-    if grep -q "or setting is" "${fx_profile}"; then
-        echo "We are patching ${fx_profile}. You will see this message only once."
-        patch -p0 <scripts/patches/firefox_profile.diff
-    fi
+    python -S -m tools.patch_selenium
```

Comment:
lol, nice way to enforce only stdlib :P

Reply:
I love how you streamlined the process! Go, Python go!